src/runtime/basic-runtime.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright 2020 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AutoPromptType} from '../api/basic-subscriptions';
+import {ConfiguredRuntime} from './runtime';
+
+const RUNTIME_BASIC_PROP = 'SWG_BASIC';
```

Comment:
Could you rename this constant to "BASIC_RUNTIME_PROP"? I think that would be more consistent with the other names in this script

Reply:
Good catch, done.